### PR TITLE
Add i18n

### PR DESCRIPTION
**Pull request by tetogomez**
### What does this PR do?
- Add i18n
- Refactor logic to download PDF

### How should this be tested?
- yarn
- yarn start
- go to `http://localhost:3000`

### CODE REVIEW

**Comment by adriexnet** [comment]
👍

File: src/language/en.json [hunk 1]
```diff
@@ -1,10 +1,46 @@
...
-    "products": "Products",
-    "productsTitle": "Products list",
-    "usersTitle": "Users list",
-    "logout": "Logout"
+    "logout": "Logout",
+    "mainTitle": "Notarize",
+    "resultPage": "Result",
+    "result": {
+      "defaultTitle": "No Title",
+      "hashTitle": "Hash del Certificado",
```
Comment (xavier506) [inline src/language/en.json]: Certificate Hash

File: src/language/en.json [hunk 2]
```diff
@@ -1,10 +1,46 @@
...
+      "date": "Date",
+      "account": "Account",
+      "titleInfo": "Title",
+      "comment": "Comment",
+      "idInfo": "Transaction ID",
+      "qrMessage": "Scan this QR code with your mobile camera to copy the hash.",
+      "donwloadButton": "Download as PDF",
```
Comment (xavier506) [inline src/language/en.json]: download typo (in key)

File: src/language/en.json (hunk 1 above)
Comment (tetogomez) [reply src/language/en.json]: done

File: src/language/en.json (hunk 2 above)
Comment (tetogomez) [reply src/language/en.json]: done

**Review by xavier506 — APPROVED** [review]

**Pull request merged**